presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -525,6 +526,11 @@ public SystemSessionProperties(
                         "Experimental: Enable statistics calculator",
                         featuresConfig.isEnableStatsCalculator(),
                         false),
+                integerProperty(
+                        MAX_TASKS_PER_STAGE,
+                        "Limits the number of tasks for a fixed distribution stage",
```

Comment:
Curious: where is the name `fixed distribution stage` from? I guess it's from `FixedSourcePartitionedScheduler` ? :)

Reply:
I aggree that the description is kinda confusing. But i coudn't come up with anything better.